### PR TITLE
Add Ehcache options; require stdlib 4.25.0

### LINKED ISSUE #245: Support for additional cluster.properties config
It'd be nice to be able to specify additional cluster config settings for the `cluster.properties` file. Such as:
* ehcache.listener.hostName
* ehcache.listener.port
* ehcache.object.port

Right now, manually adding them causes them to be overwritten on next puppet run by the template. 

Current work around is to perform initial provisioning of a Data Center node with `datacenter => true`, then set it to `false` and manually add additional config settings.

### PR DESCRIPTION
**Pull request by nrhtr**
#### Pull Request (PR) description
Add additional options for configuring the "ehcache" settings in cluster.properties (https://developer.atlassian.com/server/jira/platform/configuring-a-jira-cluster/)

#### This Pull Request (PR) fixes the following issues
Fixes #245

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -49,6 +49,9 @@
   Hash $jira_config_properties                                      = {},
   Boolean $datacenter                                               = false,
   $shared_homedir                                                   = undef,
+  $ehcache_listener_host                                            = undef,
```
Comment (bastelfreak) [inline manifests/init.pp]: can you please add datatypes for the new parameters?

**Comment by bastelfreak** [comment]
Hi @nrhtr, thanks for the PR. Can you please have a look at the used email address in the commits? It isn't associated with your github acount.

**Comment by nrhtr** [comment]
Hi @bastelfreak. Will fix up the data types shortly. I'm using my work laptop which is why it has a different email configured from my personal Github account. I can commit with my personal email configured if you prefer?

**Comment by bastelfreak** [comment]
Adding your corp mail address to github would also be fine. The used address just has to have a relation to the github account.

File: manifests/init.pp (hunk 1 above)
Comment (bastelfreak) [reply manifests/init.pp]: We can probably use this type? https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/port.pp
In this case please update the stdlib dependency in metadata.json to at least 4.25.0

Comment (nrhtr) [reply manifests/init.pp]: Added data types and bumped stdlib dependency

**Comment by bastelfreak** [comment]
Thanks!

**Pull request merged**